### PR TITLE
feat:add project and task name args

### PR DESCRIPTION
**Pull request by siewyeng**
addresses #84. Yet to update `tp-timesheet --help` and timesheet form

### CODE REVIEW

**Review by lamchuntat — APPROVED** [review]
LGTM

File: tp_timesheet/__main__.py [hunk 1]
```diff
@@ -73,6 +73,19 @@ def parse_args():
     parser.add_argument(
         "--version", action="version", version=f"%(prog)s {__version__}"
     )
+    parser.add_argument(
+        "--project",
+        type=str,
+        default="Jupiter Staffing APAC",
```
Comment (ThorpeJosh) [inline tp_timesheet/__main__.py:79]: It is difficult to type this out as a terminal option.

I think it's best to just use lower-case space less keywords like what @changukshin described in the issue discussion.

`live` `jupyter` etc. However how do we also attach the hours to it? For example we should be able to submit 4 hours to one project/task combination and 4hours to another project/task combination.

Might be better to have a set of supported options: live, OOV, holiday, training, idle, etc
And each then have an implied project/task?

Comment (ThorpeJosh) [reply tp_timesheet/__main__.py:79]: Can continue discussion in issue if needed

Comment (ThorpeJosh) [reply tp_timesheet/__main__.py:79]: Seems you addressed it in [#3da12c7](https://github.com/ThorpeJosh/tp-timesheet/pull/75/commits/3da12c725f2da98143e36322c311d88cf38fba7e)

Comment (siewyeng) [reply tp_timesheet/__main__.py:79]: OH @ThorpeJosh HAHA I can't find that anymore, maybe cos I force pushed. Okay! I think the implied project idea sounds good. I'll leave the issue of the hours out first

**Pull request merged**